sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -134,3 +136,27 @@ def _cloud_event_to_generated(cloud_event, **kwargs):
         additional_properties=cloud_event.extensions,
         **kwargs
     )
+
+def _build_request(endpoint, content_type, events, client):
+    serialize = client._serialize # pylint: disable=protected-access
+    header_parameters = {}  # type: Dict[str, Any]
+    header_parameters['Content-Type'] = serialize.header("content_type", content_type, 'str')
+
+    query_parameters = {}  # type: Dict[str, Any]
+    query_parameters['api-version'] = serialize.query("api_version", "2018-01-01", 'str')
+
+    body = serialize.body(events, '[object]')
```

Comment:
I believe you can just directly call `json.dumps(events)` instead of first serializing the body with the msrest serializer

Reply:
single events are not JSON serializable - so need this